src/style/style.scss

```diff
@@ -0,0 +1,7 @@
+.modal{
+  width: 25%;
+  margin-left: auto;
+  margin-right: auto;
+  border: 4px solid #0B456F;
+  padding: 30px;
+}
```

Comment:
please, remove that file:
1. it should be at the project, but into the another place
2. modal is not included into your ticket